service/stacks/zephyr/sal_hfp_ag_interface.c

```diff
@@ -394,6 +405,50 @@ bt_status_t do_ag_disconnect(bt_controller_id_t id, bt_address_t* addr, void* us
     return BT_STATUS_SUCCESS;
 }
 
+void do_ag_sco_connect(service_work_t* work, void* userdata)
+{
+    ag_connect_sco_params_t* params = (ag_connect_sco_params_t*)userdata;
+    if (!params || !params->ag) {
```

Comment:
we could make codes nice , likes :
if (!params) {
  return;
  }
  
  if(!params->ag) {
    free(params);
    return;
    }